opm/common/OpmLog/EclipsePRTLog.hpp

```diff
@@ -29,16 +29,21 @@ namespace Opm {
 class EclipsePRTLog : public StreamLog {
 
 public:
-    using StreamLog::StreamLog;
 
     void addTaggedMessage(int64_t messageType, const std::string& messageTag, const std::string& message);
 
     size_t numMessages(int64_t messageType) const;
 
     ~EclipsePRTLog();
 
+    EclipsePRTLog(const std::string& logFile , int64_t messageMask,
```

Comment:
I know this class was already missing documentation, so it's a bit unfair to ask you to provide it, still: it would be good to document the constructors, at least the new argument.